### DISCUSSION TITLE
「VR眼鏡」,你真的了解嗎?

### DISCUSSION THREAD

**Discussion by titan92601** [1101-GS4719-C/Python-Community-GS4719-C#1949] [科技新知]
VR(Virtual Reality)，亦即虛擬實境。利用計算機技術創建模擬環境以及感官，例如視覺、聽覺、觸覺，甚至嗅覺，讓用戶在3D世界上進行交互。VR需要包含三大元素，3D空間、實時交互、自身投影。而VR的基本技術原理利用了人工智能(Artificial Intelligence)以及機械學習(Machine Learning)、並利用數據分析(Data Analysis)來判斷用家的真實動作。

VR眼鏡是一種頭戴式顯示設備。原理是將小型2D顯示器所產生的影像藉由光學系統放大，左右眼的顯示屏都是一塊眼鏡鏡片的尺度，可以提供長達20米的視覺成像距離，用左右眼分別看見不一樣的分割畫面，使雙眼將看見的不同圖像自動疊加後，眼前影像就會產生立體、塑造出逼真的三維空間感，讓人彷彿親身進入虛擬世界一般，達到沉浸式的感官體驗。1968年Ivan Sutherland在他的學生Bob Sproull 的幫助下，創建了第一個VR及AR頭戴式顯示系統。而市面上常見的VR眼鏡主要又可分為兩類：PCVR、一體機，接下來就為大家一一介紹，幫助你釐清他們的差別。

PCVR顧名思義，即是需要仰賴PC（電腦）做即時運算的VR眼鏡，因此除了裝置本身，還需搭配轉換器、電源供應器、USB傳輸線等使用，像Valve Index、VIVE Cosmos Elite、VIVE Pro 2就屬於PCVR。而PCVR的優點是玩遊戲時可仰賴PC本身強大的圖形運算能力，因此更能順暢體驗fps、畫面細緻度都高的3A遊戲，不過這個先決條件還是要搭配的硬體有一定的規格才能成立，因此若想要玩場面逼真的動作類型遊戲，其CPU、GPU、記憶體等都要到位方能讓VR體驗流暢進行。像Steam上就提供VR性能測試，使用者可以透過此方式檢測自己的PC是否達到可運行VR遊戲的標準（若不符合，就需多點預算升級裝備才能享受VR遊戲惹！）

此外，PCVR的一個痛點，就是必須接線使用，如此戴著VR眼鏡時的移動範圍就會限縮，且PCVR是採用基地台追蹤定位，因此需要額外在使用空間的周圍進行部署，前置作業比較繁瑣一點，但優點是定位相當精準，現實中的一舉一動都能如實在虛擬世界中呈現，讓遊戲體驗更加分。而隨著VR眼鏡的研發持續進行，也出現了「無線模組」的輔助裝置，可以讓使用者與PCVR一同使用，擺脫線材的束縛，提升遊玩的便利性。

一體機的設備本身即搭載系統晶片（SoC, System on a Chip），其中就包含GPU、CPU並支援5G，而目前主流的VR一體機晶片都是用高通的XR2，且採用Inside-Out追蹤技術，不必裝設基地台就能精準定位，只要戴上就能玩VR遊戲，不用再接許多線路連PC，隨時隨地都可使用，外出想帶個遊戲機打發時間或與親友同樂，只要帶著一台一體機就能搞定，相當便利。而隨著VR技術日漸成熟，鏡片也從原先傳統的菲涅耳透鏡（Fresnel Lens）改良成輕薄的餅乾透鏡（Pancake Lens），設備中的晶片也越做越小，朝向輕量化的方向發展，像是Quest Pro、VIVE XR Elite都是在此一浪潮下所誕生的機型。

不過也因為一體機僅能用本體運算，相較於用電腦運算的PCVR，晶片效能再強也比不上電腦，因此一體機比較適合玩畫面細緻度要求不高的遊戲。但如果真的想玩3A大作等級或僅有PC版的VR遊戲怎麼辦？使用一體機最彈性的地方就在於，可以透過無線串流的方式連上PC，一體機不僅可以瞬間變成PCVR，還能享受無線的自由，這樣變動自如的特性，真的很方便阿！

VR應用的產業能夠應用的有很多，包括醫學、軍事航天、室內設計、工業仿真等等，其中成人娛樂產業亦得益甚多，「VR Porn」的搜尋次數一直居高不下，在香港每月有6000次搜尋次數。

如果你還好奇其它關於使用VR眼鏡的相關問題或使用心得可以參考以下連接
https://www.vivepostwave.com/9880/vr-glasses-and-vr-headset/

**Comment by YANRONG0123** [comment]
隨著科技越來越進步，相關靈活運用的工具也會隨之遽增，像是此文章中提到的VR眼鏡，以目前的技術來說，還需要連接線路才能夠進行，也許未來就可以利用其他供電貨供應資源的方式，讓我們看看結合虛擬與現實的世界，使我們的生活更加便利，我認為科技有所突破絕對是一件再好不過的事情。

**Comment by hbh10601** [comment]
以前在朋友家玩過VR眼鏡，那時候他家的有線版真的很限縮、會有卡卡的感覺，有時候手揮一揮還會揮到線路，然後就被拉回現實，不過那是好幾年前的事情了，看到這篇讓我有想要再去試試看現在無線版本的VR眼鏡。

**Comment by yxin214** [comment]
大約4年前，我曾去過VR主題樂園，那是一次很棒的體驗！我玩了泛舟、空中腳踏車等設施，體驗十分真實。我想唯二的缺點就是門票昂貴和VR眼鏡太重了，導致有「異物感」。票價昂貴是必然，畢竟一項新技術是投入許多資金開發而成，但我相信，若能克服VR眼鏡的異物感，VR在未來會有更多受眾，並在更多領域加以應用。

**Comment by yyujimmy** [comment]
對我而言，我對一體機的概念很感興趣，因為它不需要依賴外部設備，讓我能夠更自由地體驗虛擬世界，而不必擔心線材的束縛或設置基地台。這讓VR的應用更具靈活性，可以隨時隨地進行，尤其是對於想要在不同地方使用VR的人來說非常方便。

此外，文章提到了VR技術在不同領域的應用，例如醫學和工業仿真，這些應用展示了VR技術的巨大潛力，不僅僅是作為一種娛樂工具，更是一種強大的工具，可以在許多不同的領域產生實際影響。

**Comment by KH1126** [comment]
VR眼鏡可讓用戶感受到虛擬現實世界，藉由使用AR眼鏡可提供沉浸式的虛擬體驗。
最近在觀看香格里拉・開拓異境後，
感覺以後如果也可以使用AR眼鏡來達成如此逼真遊戲效果的話感覺超讚。
